manifests/params.pp

```diff
@@ -40,8 +40,8 @@
                           }]
 
   $host            =     '0.0.0.0'
-  $port            =     3000
+  $port            =     '3000'
   $user            =     ''
   $pass            =     ''
-  $refresh         =     5
+  $refresh         =     '5'
```

Comment:
I agree that this is confusing, but in many cases (non-hiera) the thing ends up being a string, so I do think we (Yelp) would needs the second everyone-happy case.